src/test/java/uk/gov/digital/ho/hocs/casework/client/documentclient/DocumentClientTest.java

```diff
@@ -0,0 +1,112 @@
+package uk.gov.digital.ho.hocs.casework.client.documentclient;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.digital.ho.hocs.casework.application.RestHelper;
+
+import java.util.*;
+
+import static java.util.UUID.randomUUID;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class DocumentClientTest {
+
+
+    @Mock
+    private RestHelper restHelper;
+
+    private DocumentClient documentClient;
+    private final String caseType = "MIN";
+    private final UUID caseUUID = randomUUID();
+
+    private String documentService = "http://document-service";
+    ;
+
+    @Before
+    public void setUp() {
+        documentClient = new DocumentClient(restHelper, documentService);
+    }
+
+    @Test
+    public void getDocuments() {
+        GetDocumentsResponse mockedResult = mock(GetDocumentsResponse.class);
+        String url = "/document/reference/" + caseUUID.toString();
```

Comment:
It's best to create actual instances of GetDocumentResponse, DocumentDto and S3Document in these tests. The actual instance values can then be checked after the client call.
An example of this if the Http Headers instantiation in getFile of RestHelperTest.